doc/api/next_api_changes/removals/30005-DS.rst

```diff
@@ -0,0 +1,11 @@
+``matplotlib.cm.get_cmap``
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Colormaps are now available through the `.Coloarmap.Registry` accessible via
```

Comment:
Sorry, typo in my propsal that made the doc build fail.

```suggestion
Colormaps are now available through the `.ColormapRegistry` accessible via
```
I take the liberty to commit